src/IsaacApiTypesAutoGenerated.tsx

```diff
@@ -445,7 +472,6 @@ export interface RegexPatternDTO extends ChoiceDTO {
 }
 
 export interface SeguePageDTO extends ContentDTO {
-  canonicalSourceFile?: string;
```

Comment:
are these moving around? I guess it's the linter changing the order of things, but this one appears to have moved from one interface to another

Reply:
Sort of, it's just down to how the api plugin is generating the types. Now that I've rebased that change is gone!